libraries/rush-lib/src/utilities/performance.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import type { PerformanceEntry, PerformanceMark, PerformanceMeasure } from 'node:perf_hooks';
+
+/**
+ * Starts a performance measurement.
+ * @param name - The name of the performance measurement. This should be unique for each measurement.
+ * @returns The performance mark that indicates the start of a measurement.
+ */
+export function startPerformanceMeasurement(name: string): PerformanceMark {
+  return performance.mark(`${name}:start`);
+}
+
+/**
+ * Ends a performance measurement.
+ * @param name - The name of the performance measurement. This should match the name used in `startPerformanceMeasurement`.
+ * @returns The performance measure that contains the start and end marks for the measurement.
+ */
+export function endPerformanceMeasurement(name: string): PerformanceMeasure {
+  return performance.measure(name, `${name}:start`);
```

Comment:
```suggestion
  return performance.measure(name, `${name}:end`);
```

Reply:
The second argument to measure is the name of a start mark to look up.